wandb/sdk/wandb_init.py

```diff
@@ -466,7 +466,7 @@ def init(self) -> Union[Run, Dummy, None]:  # noqa: C901
             run._set_run_obj(ret.run)
 
         # initiate run (stats and metadata probing)
-        _ = backend.interface.communicate_run_start()
+        _ = backend.interface.communicate_run_start(run)
```

Comment:
Uhhhh, not sure why we would need this now but not before...

Reply:
This is how the resume information is conveyed now that the step handling is happening in handler instead of sender

Reply:
it is a pretty clean way of synchronizing the run_object (and hence resume state) between user process and internal process.

It was not used before because we didnt really need the run_object back in the handler, though it was part of the plans, it was a bug that it wasnt being sent.